src/redux/categories/categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Categories() {
+  return <div>Under construction</div>;
+}
+
+export default Categories;
```

Comment:
- [ ] This file should contain your categories actions and reducer and not the Categories component, you already have that in the /src/components/Categories.js. and as requested by the previous reviewer, you should define an action type for checking the status, set the initial state to be an empty array of categories, export Action Creators for your actions, write your reducer and export it as default, and the check status action should return a string "Under construction".